pymc/backends/__init__.py

```diff
@@ -32,18 +32,18 @@
 To discard the first N values of each chain, slicing syntax can be
 used.
 
-    >>> trace['x', 1000:]
+    >>> trace["x", 1000:]
 
 The `get_values` method offers more control over which values are
 returned. The call below will discard the first 1000 iterations
 from each chain and keep the values for each chain as separate arrays.
 
-    >>> trace.get_values('x', burn=1000, combine=False)
+    >>> trace.get_values("x", burn=1000, combine=False)
```

Comment:
Is this still called burn?

Reply:
It looks like it yes

https://github.com/pymc-devs/pymc/blob/79fafb04bd5e26bf1e7057e80860107aa5939c1a/pymc/backends/base.py#L438-L441